check_sites.py

```diff
@@ -100,16 +111,19 @@ def main():
                     "\nerror was: " + inst)
                 return
         old_checker = FpsCheck(old_sites, etlds, icanns)
-        check_sets, _ = find_diff_sets(old_checker.load_sets(), fps_checker.load_sets())
+        check_sets, subtracted_sets = find_diff_sets(old_checker.load_sets(), fps_checker.load_sets())
         # TODO: add variable and check for subtracted_sets in case of user 
         # removing old set from the list
     else:
         check_sets = fps_checker.load_sets()
 
+    # Run check on subtracted sets
+    if subtracted_sets:
+        fps_checker.find_invalid_removal(subtracted_sets)
```

Comment:
```suggestion
    fps_checker.find_invalid_removal(subtracted_sets)
```

The check itself should be a no-op if the dict arg is empty, so we don't have to skip it in that case. It makes things clearer if the caller code can be straight-line, and rely on the checks themselves to do as much or little work as they need to.